nucypher/crypto/kits.py

```diff
@@ -76,3 +80,39 @@ def __init__(self, *args, **kwargs) -> None:
     def from_bytes(cls, some_bytes):
         capsule, sender_pubkey_sig, ciphertext = cls.split_bytes(some_bytes)
         return cls(capsule=capsule, sender_pubkey_sig=sender_pubkey_sig, ciphertext=ciphertext)
+
+
+class RevocationKit:
+
+    def __init__(self, policy_treasure_map, signer: 'SignatureStamp'):
```

Comment:
Perhaps put a `self.__confirmations` dict here?

Reply:
I'd like to leave confirmations for another PR. Probably the same one that will deal with failed revocations.

Reply:
Sounds like good scoping. In that case I think it deserves an Issue.

Reply:
See #569

Reply:
:+1:

Reply:
(adding this number to the TODOs is a nice touch)